Gemfile.lock

```diff
@@ -10,6 +10,28 @@ PATH
 GEM
   remote: http://rubygems.org/
   specs:
+    actionmailer (6.0.3.4)
```

Comment:
I hope to see almost no changes here in the Gemfile.lock once you back out `exception_handling`.